app/src/main/java/waiter/Router.java

```diff
@@ -1,23 +1,31 @@
 package waiter;
 
+import java.util.AbstractMap;
 import java.util.Map;
 
 public class Router {
 
     private final Map<String, Map<String, Map<String, String>>> routes;
     private final String noMethod = "No Method";
     private final String noUrl = "No Url";
+    Map.Entry<String, Map<String, String>> simpleGet = new AbstractMap.SimpleEntry<>("/simple_get", setResponseFields("200 OK", "Content-Length:", ""));
+    Map.Entry<String, Map<String, String>> noUrlFound = new AbstractMap.SimpleEntry<>(noUrl, setResponseFields("500 Internal Server Error", "Content-Length:", ""));
 
     public Router() {
         this.routes = Map.ofEntries(
                 Map.entry("GET", Map.ofEntries(
-                        Map.entry("/simple_get", setResponseFields("200 OK", "Content-Length:", "")),
+                        simpleGet,
                         Map.entry("/simple_get_with_body", setResponseFields("200 OK", "Content-Length:", "Hello world")),
```

Comment:
Any reason we inline this route but not the others?

Reply:
The inlined ones were not used repeatedly so I didn't think pulling them out would be necessary (open to change though)